.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,18 +8,15 @@ Fixes #
 
 *
 
-<!-- Add the following only if there are new/updated actions or filters. Please provide a brief description of what they do and any arguments they may take. Be sure to also add the "Hooks" label to this PR. -->
-### New/Updated Hooks
-
-*
-
-<!-- Add the following only if there is any code that is being deprecated. Please list the replacement function or hook that should be called instead, if applicable. Be sure to also add the "Deprecation" label to this PR. -->
-### Deprecated Code
-
-*
-
 <!--
 Helpful tips for screenshots:
 https://en.support.wordpress.com/make-a-screenshot/
 -->
 ### Screenshot / Video
+
+### Checklist
+- [ ] My code is tested.
+- [ ] I've set a milestone.
+- [ ] I've added a brief description for new/updated actions and filters and applied the "Hooks" label.
+- [ ] I've listed template files that have changed and applied the "Templates" label.
+- [ ] I've listed code that has been deprecated and applied the "Deprecation" label. <!-- Please list the replacement function or hook that should be called instead, if applicable. -->
```

Comment:
Cool! I'd suggest also add here a check that the deprecated methods were added to the respective issue to be removed.

Reply:
Why do we need the issue which has all the methods in the first place? Aren't we going to search for deprecations in the code anyway in case something was forgotten?

Reply:
I think the issues are just to make this work easier when removing the methods. But yeah, if we check the issues by deprecation label one by one, it's double work. Also, we'll probably find the deprecations through the code when doing this task (probably the easiest way to do it).

Reply:
I currently use these labels to write the release post to ensure developers are aware of any new or changed hooks, deprecated code or template file changes. See [this post](https://senseilms.com/2021/01/14/sensei-lms-3-7-woocommerce-paid-courses-2-3-beta-releases/) as an example. I currently filter PRs by label to know what to add to the release notes and ensure nothing is missed.

Reply:
I think @gkaragia meant about the issues, like these: https://github.com/Automattic/sensei/issues/2407 https://github.com/Automattic/sensei/issues/2976